common/urls.py

```diff
@@ -30,6 +30,7 @@
     url(r'^selectiongproteinpredefined', views.SelectionGproteinPredefined, name='selectiongproteinpredefined'),
     url(r'^selectiongproteintoggle', views.SelectionGproteinToggle, name='selectiongproteintoggle'),
     url(r'^targetformread', views.ReadTargetInput, name='targetformread'),
+    url(r'^biastargetformread', views.ReadBiasTargetInput, name='biastargetformread'),

```

Comment:
indeed it is almost the same except last part for fetching protein by ID, which I need. i wondered if I should append 
if obj == None and (up_name.isnumeric()):
            selection_subtype = 'protein'
            try:
                obj = up_name
            except:
                obj = None
to the ReadTargetInput  as additional option or create new one

Reply:
I think a clean way could be to just modify the addTarget and removeTarget functions in JS to use the "id" instead of the "entry-value". That way you are using the slug in your submission to targetFormRead, which is supported by default.